storage/store.go

```diff
@@ -0,0 +1,39 @@
+package storage
+
+import (
+	"context"
+	"sync"
+)
+
+type Store struct {
+	mu sync.RWMutex
+	// highest block height
+	blockHeight uint64
```

Comment:
a comment suggests a better naming: `latestHeight`

Reply:
Good point :+1: 
Updated in https://github.com/onflow/flow-evm-gateway/pull/11/commits/ed6023163a95fab26b79876d016e8de491ec1de6